index.js

```diff
@@ -15,12 +17,18 @@ function azurestore(config){
 
 azurestore.prototype.save = function (image) {
     var fileService = azure.createBlobService(options.connectionString);
-    var uniqueName = new Date().getMonth() +"/" + new Date().getFullYear()+"/"+ image.name;
-    return nodefn.call(fileService.createContainerIfNotExists.bind(fileService), options.container, {publicAccessLevel: 'blob'})
+    var uniqueName = new Date().getMonth() + "/" + new Date().getFullYear() + "/" + image.name;
+    return nodefn.call(fileService.createContainerIfNotExists.bind(fileService), options.container, { publicAccessLevel: 'blob' })
     .then(nodefn.call(fileService.createBlockBlobFromLocalFile.bind(fileService), options.container, uniqueName, image.path))
     .delay(500) //todo: this was placed here per issue #4 (aka sometimes things 404 right after upload) figure out a better way than just adding a delay
-    .then(function(){
-        return fileService.getUrl(options.container, uniqueName);
+    .then(function () {
+        var urlValue = fileService.getUrl(options.container, uniqueName);
+
+        if (!options.cdnUrl)
+            return urlValue;
+
+        var parsedUrl = url.parse(urlValue, true, true);
+        return "https://" + options.cdnUrl + parsedUrl.path;
```

Comment:
I don't think we should hardcode the protocol here (http/https) as some people may want to specify one over the other. 

perhaps instead we could have an option that is called useHttps which defaults to false, and if people want https they can turn it on.